src/main/app/src/app/taken/taken-werkvoorraad/taken-werkvoorraad.component.ts

```diff
@@ -237,4 +255,61 @@ export class TakenWerkvoorraadComponent
     this.selection.clear();
     this.dataSource.filtersChanged();
   }
+
+  private handleAssigmentOrReleasementWorkflow<T>(
```

Comment:
```suggestion
  private handleAssignOrReleaseWorkflow<T>(
```

Reply:
Or could this be created as 2 named types (aliases?) for the same underlying type? That would even be nicer IMHO 😸